vcpkg/github-integration.md

```diff
@@ -0,0 +1,94 @@
+
+Repositories that use GitHub Actions for continuous integration can take advantage of [vcpkg's binary caching feature](./users/binarycaching.md) to reduce the amount of time it takes to build their projects. There are two binary cache providers that are available to GitHub repositories.
+
+### The GitHub Actions cache provider
+
+[!INCLUDE [experimental](../includes/experimental.md)]
+
+The [GitHub Actions cache](https://docs.github.com/en/actions/using-workflows/caching-dependencies-to-speed-up-workflows) is intended to store a repository's intermediate build files that don't change often between jobs or workflow runs. GitHub provides a few different [tools to manage your Actions caches](https://docs.github.com/en/actions/using-workflows/caching-dependencies-to-speed-up-workflows#managing-caches), which include REST APIs and an extension to the gh command line tool, so that you can optimize caches for your workflow. vcpkg's integration with GitHub Actions cache is through the [`x-gha` binary source provider](./users/binarycaching.md#gha).
+
+### The GitHub Packages NuGet cache provider
+
+[GitHub Packages](https://docs.github.com/en/packages/learn-github-packages/introduction-to-github-packages) allows a repository to publish binary artifacts for public or private use. In addition to hosting storage for published packages, GitHub Packages supports a [variety of package management tools](https://docs.github.com/en/packages/learn-github-packages/introduction-to-github-packages#supported-clients-and-formats) by acting as a package registry. vcpkg can use the NuGet registry interface to GitHub Packages as a cache for vcpkg binary artifacts, by using the [`nuget` binary source provider](./users/binarycaching.md#nuget).
+
+## The GitHub dependency graph
+
+[!INCLUDE [experimental](../includes/experimental.md)]
+
+> [!TIP]
+> We are actively developing this feature and would like to hear your feedback. Let us know if you have any thoughts about the current functionality or any desired functionality by emailing [vcpkg@microsoft.com](mailto:vcpkg@microsoft.com) or by filing an issue in [vcpkg-docs](https://github.com/microsoft/vcpkg-docs/issues). If you have feedback on any of the GitHub features that this enables, let us know that too and we'll make sure it gets to the right folks at GitHub.
+
+### About the GitHub dependency graph
+
+The GitHub [dependency graph](https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/about-the-dependency-graph) stores the set of dependencies for a repository. Beyond being able to visualize a repository's dependencies, GitHub builds several useful features on top of this data, including [dependency review](https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/about-dependency-review) and [Dependabot alerts](https://docs.github.com/en/code-security/dependabot/dependabot-alerts/about-dependabot-alerts).
```

Comment:
This is a CPub style thing: we don't inline links in a sentence. We explain the thing and then use: "For more information, see [link]". 

I've wondered about this rule and I'm not sure why it exists. I wonder if inline links may make sense in English but may not when translated because the link is embedded in an order specific to English grammar? Or maybe they make it hard for the visually impaired because perhaps the screen reader does a weird thing in the middle of the sentence it is reading?

Whatever the reason, our style is to explain a thing and then say "For more information, see [the link]()."
We have a guide that you may find helpful if you do a bunch of docs writing that's built on lessons learned from writing for large audiences. For more information about our writing style, see [cpub content guide](https://review.learn.microsoft.com/help/get-started/?branch=main&tabs=learn).

Reply:
Interesting. Every accessibility guideline I've read before argues for the opposite (the way I've done it here). That a hyperlink that it attached to something like `click <a href="blah>here</a>` becomes unusable by screen readers.

Sampling:
https://www.fullerton.edu/ati/website/best-practices-accessible-links.php
https://www.nysed.gov/webaccess/create-accessible-hypertext-links
https://accessibility.princeton.edu/how/content/links

The WCAG guidelines aren't exactly clear, but I think they point towards the approach that I've used here. https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=244#qr-navigation-mechanisms-refs

Reply:
I've made changes to try to adhere to this style guide.